doc/api/events.markdown

```diff
@@ -162,3 +162,33 @@ added.
 This event is emitted *after* a listener is removed.  When this event is
 triggered, the listener has been removed from the array of listeners for the
 `event`.
+
+### Inheriting from 'EventEmitter'
+
+Inheriting from `EventEmitter` is no different from inheriting from any other
+constructor function. For example,
```

Comment:
I see that example code is [usually](https://github.com/nodejs/io.js/blob/master/doc/api/fs.markdown#fsrealpathpath-cache-callback) [introduced](https://github.com/nodejs/io.js/blob/master/doc/api/http.markdown#responsewriteheadstatuscode-statusmessage-headers) with a colon, not a comma.